frontend/src/components/AccountSentTable.vue

```diff
@@ -1,97 +1,127 @@
 <template>
-  <q-table
-    :grid="$q.screen.xs"
-    card-container-class="col q-col-gutter-md"
-    :columns="mainTableColumns"
-    :rows="formattedSendMetadata"
-    :no-data-label="$t('AccountSendTable.account-empty')"
-    :pagination="paginationConfig"
-    row-key="dateSentUnix"
-    :title="$t('AccountSendTable.sent-funds')"
-  >
-    <template v-slot:top-left="props">
-      <div class="flex">
-        <div class="q-table__title" :props="props">{{ $t('AccountSendTable.sent-funds') }}</div>
-        <base-tooltip class="q-ml-sm self-center" size="xs" icon="fas fa-question-circle">
-          {{ $t('AccountSendTable.storage-description') }}
-        </base-tooltip>
-      </div>
-    </template>
-    <!-- Card Layout for grid option -->
-    <template v-slot:item="props">
-      <div :key="props.row.dateSentUnix" class="col-12">
-        <q-card class="card-border cursor-pointer q-pa-md col justify-center items-center">
-          <q-card-section class="row justify-center items-center">
-            <img class="q-mr-md" :src="props.row.tokenLogo" style="width: 1.2rem" />
-            <div class="text-primary text-h6 header-black q-pb-none">
-              {{ props.row.amount }} {{ props.row.tokenSymbol }}
-            </div>
-          </q-card-section>
-          <div class="row justify-between items-center">
-            <div>
-              <span class="q-mr-xs">{{ $t('AccountSendTable.receiver') }}</span>
-            </div>
-            <div>
-              <div @click="copyAddress(props.row.address, provider)" class="cursor-pointer copy-icon-parent">
-                <span>{{ props.row.addressShortened }}</span>
-                <q-icon color="primary" class="q-ml-sm" name="far fa-copy" />
+  <div>
```

Comment:
_(arbitrary comment location to start a thread)_

I found it unintuitive that the tx flow was approve tx -> signature -> send tx. I understand why, but I think we should put the signature first

Reply:
Changed order https://github.com/ScopeLift/umbra-protocol/pull/494/commits/ccb1838d7f3aacacaed8d5551c65f8f119e70d03